src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/JDBCEncryptionDecryptionTest.java

```diff
@@ -134,7 +132,7 @@ public void testBadAkv(String serverName, String url, String protocol) throws Ex
 
         try {
             SQLServerColumnEncryptionAzureKeyVaultProvider akv = new SQLServerColumnEncryptionAzureKeyVaultProvider(
-                    (SQLServerKeyVaultAuthenticationCallback) null);
+                    (TokenCredential) null);
```

Comment:
Again, we are not removing the SQLServerKeyVaultAuthenticationCallback anymore, so keep this test as it is and add a test for TokenCredential separately.